rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- Please move **book** and **person** to `attr_accessor`. You can refer to the same explanation and the link I noticed in book class